### ISSUE TITLE
kubectl not found on k8smaster

### ISSUE THREAD

**Issue by celeguim** [0zores/k8s-vagrant#3]
[vagrant@k8smaster ~]$ hostname
k8smaster
[vagrant@k8smaster ~]$ kubectl get nodes
-bash: kubectl: command not found
[vagrant@k8smaster ~]$

**Comment by 0zores** [comment]
Hmmm strange, I created a new branch called "develop" where I included one option to increase the verbosity, can you clone the repo and checkout the develop branch, run again and send to me the output pls

**Comment by celeguim** [comment]
Yes it's strange, and stranger is that even the master branch is working fine now, it's probably something in my box that I didn't observe before. So thank you and congrats for this nice project.
